app/src/styles/fileList.js

```diff
@@ -71,9 +71,6 @@ const fileListStyle = StyleSheet.create({
   },
   progress: {
     marginTop: (screenWidthPixels <= 720) ? 2 : 4,
-    height: 3,
```

Comment:
Outside of this PR, but if 720 is a special value that appears in multiple places it should be a constant.